src/CheckIn.js

```diff
@@ -599,7 +599,7 @@ class CheckIn extends React.Component {
                   </Col>
                   <Col xs={12} sm={1}>
                     <Layout className="marginTopLabelSpacer">
-                      <Button id="clickable-end-session" buttonStyle="default" style={{ minWidth: '90px' }} fullWidth onClick={this.handleSessionEnd}>
+                      <Button id="clickable-end-session" buttonStyle="default" style={{ minWidth: '200px' }} fullWidth onClick={this.handleSessionEnd}>
```

Comment:
Could you please put relevant JIRA issue (https://issues.folio.org) in the issue title and in `## Refs` section in pull request?
Could you please update CHANGELOG.md?
The current fix is breaking the grid layout (https://getbootstrap.com/docs/4.0/layout/grid/) grid.JPG.
![grid](https://user-images.githubusercontent.com/24813219/117117646-77b64400-ad98-11eb-8abd-f90673d4d33a.JPG)

Could you please check should we change grid layout here instead of changing button width?